lib/config/common.ts

```diff
@@ -52,7 +52,7 @@ export interface RenovateSharedConfig {
   repositoryCache?: RepositoryCacheConfig;
   requiredStatusChecks?: string[];
   schedule?: string[];
-  semanticCommits?: boolean;
+  semanticCommits?: 'auto' | 'enabled' | 'disabled';
```

Comment:
Are we using this anywhere else? Perhaps make it a type?

Reply:
Do you mean the triple pattern of `auto/enabled/disabled`?

Reply:
OK